Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestManager.java

```diff
@@ -1254,29 +1265,32 @@ private void doRecoveryIfCrashed(Manifest manifest, AutoIngestJobNodeData nodeDa
                         }
 
                         /*
-                         * Update the coordination service node for the job. If
-                         * this fails, leave the recovery to another host.
+                         * Update the coordination service manifest node for
+                         * the job. If this fails, leave the recovery to
+                         * another host.
                          */
                         try {
-                            updateCoordinationServiceNode(job);
+                            updateCoordinationServiceManifestNode(job);
                             if (numberOfCrashes <= AutoIngestUserPreferences.getMaxNumTimesToProcessImage()) {
                                 newPendingJobsList.add(job);
                             } else {
-                                newCompletedJobsList.add(new AutoIngestJob(nodeData));
+                                newCompletedJobsList.add(new AutoIngestJob(jobNodeData));
                             }
                         } catch (CoordinationServiceException ex) {
                             SYS_LOGGER.log(Level.SEVERE, String.format("Error attempting to set node data for %s", manifestPath), ex);
                             return;
                         }
 
                         /*
-                         * Write the alert file and do the logging.
+                         * Update the case node data and do the logging.
                          */
                         if (null != caseDirectoryPath) {
                             try {
-                                AutoIngestAlertFile.create(nodeData.getCaseDirectoryPath());
-                            } catch (AutoIngestAlertFileException ex) {
-                                SYS_LOGGER.log(Level.SEVERE, String.format("Error creating alert file for crashed job for %s", manifestPath), ex);
+                                CaseNodeData caseNodeData = new CaseNodeData(coordinationService.getNodeData(CoordinationService.CategoryNode.CASES, caseDirectoryPath.toString()));
+                                caseNodeData.setErrorsOccurred(true);
+                                updateCoordinationServiceCaseNode(caseNodeData, caseDirectoryPath);
```

Comment:
These three lines of code are duplicated many times in this class. Please create a method for them. Perhaps we don't need  updateCoordinationServiceCaseNode.